### DISCUSSION TITLE
Stamina mod or whatever

### DISCUSSION THREAD

**Discussion by GDcheerios** [ppy/osu#18359] [Ideas]
I don't really know the best name for the mod so stamina mod would work I guess and also I don't think this would work as a mod either with the new scoring system so I don't really know what to call it, but basically the idea is that you have a sudden death like rule applied and the level repeats and gets faster as you go on until you miss one then the level would finish and that would be your score

**Comment by GDcheerios** [comment]
Maybe it could be an unranked mod or even a seperate leaderboard, like relax leaderboards on other servers

**Reply by integerrr** [reply]
putting the mod proposal aside, fyi there won't be such things as "unranked mods" anymore apart from autoplay, all mods are ranked including relax for example

**Reply by GDcheerios** [reply]
oh, because automation mods have yet to get their score modifiers set?
